llvm/test/CodeGen/X86/hoist-loads-stores-with-cf.ll

```diff
@@ -0,0 +1,86 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc < %s -mtriple=x86_64 -mattr=+cf | FileCheck %s
+
+;; Test masked.load/store.v1* is generated in simplifycfg and not falls back to branch+load/store in following passes.
```

Comment:
Drop extra `;` here and below.

Reply:
The extra `;` here is on purpose to distinguish comments and run line. We can see a lot of examples

```
grep -r ";;" --include=\*.ll
```